guide/designing-products/personal-finance.md

```diff
@@ -113,14 +115,13 @@ For more on this use case, see the [daily spending case study]({{ '/guide/case-s
    layout = "float-left"
 %}
 
-Rent, utilities, subscriptions, car payments, grocery bills, and similar expenses need to be paid regularly on a weekly or monthly basis. Compared to day-to-day discretionary spending, these payments are higher in value, less frequent, and play a more important role in our lives. We can go without an extra coffee in the afternoon, but not paying rent or water bills is typically not an option.
+Higher value payments such as rent, utilities, car payments, and groceries are seldom paid for in cash. Instead, they are tied to service providers with ingrained payment mechanisms such as checks, debits, ACH, and wire transfers.
```

Comment:
```suggestion
Higher value payments such as rent, utilities, car payments, and groceries are seldom paid for in cash. Instead, they are tied to service providers with ingrained payment mechanisms such as checks or bank transfers.
```

Reply:
More concise and less US centric.
(I don't know what 'a debit' is, I do know what ACH is but only because I have lived in the US)